.gitignore

```diff
@@ -105,3 +105,6 @@ sftp-config.json
 logs/
 
 .vscode
+newman_data.json
+newman/
+dev.json
```

Comment:
Adding them here is one thing, but these should be deleted by the code, Better is to create them somewhere not in git's tracked folders like `/tmp/`

Reply:
Deleting newman_data.json now. dev.json is the configuration that devs will use when running tests and newman/  directory contains the result dashboard. I think both make sense to be kept in the postman_tests folder.